### PR TITLE
Update infos and code to use the embedded Emscripten libjpeg

### PR DESCRIPTION
**Pull request by kalwalt**
This PR is based on the issue #6 
As said libjpeg is embedded into Emscripten core lib so is not necessary to download the libjpeg and build the code. though i't's necesssary some changes in the code and in the emcc command to build the js code. I will also make some comment in the code.
The most important thing is to add the `-s USE_LIBJPEG` flag and remove the link library. It is required to change some path for the libjpeg include inside **jpeg-read.c** and **jpeg-write.c**. 
`EXTRA_EXPORTED_RUNTIME_METHODS` is deprecated in newer Emscripten that's why `EXPORTED_RUNTIME_METHODS`. 
I added the kuva.jpg image found in the old jsartoolkit5 project, but you can use every image you want.
Other infos: build with emcc 2.0.20 under Ubuntu 20.04.

### CODE REVIEW

File: jpeg-read.c [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include "libjpeg/jpeglib.h"
+#include <jpeglib.h>
```
Comment (kalwalt) [inline jpeg-read.c:4]: This because libjpeg is in the Emscripten core.

File: jpeg-write.c [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include "libjpeg/jpeglib.h"
+#include <jpeglib.h>
```
Comment (kalwalt) [inline jpeg-write.c:4]: same here.

**Comment by kalwalt** [comment]
@AntoineViau have you checked this?

**Comment by AntoineViau** [comment]
Thanks for PR, and sorry for the delay: I was was on holydays :)

**Comment by kalwalt** [comment]
> Thanks for PR, and sorry for the delay: I was was on holydays :)

NO problem! it's a pleasure!

**Pull request merged**